src/guide/template-syntax.md

```diff
@@ -16,7 +16,7 @@ The most basic form of data binding is text interpolation using the "Mustache" s
 <span>Message: {{ msg }}</span>
 ```
 
-The mustache tag will be replaced with the value of the `msg` property on the corresponding data object. It will also be updated whenever the data object's `msg` property changes.
+The mustache tag will be replaced with the value of the `msg` property from the corresponding component instance. It will also be updated whenever the `msg` property changes.
```

Comment:
I tweaked this because:

1. We haven't gone into detail about `data` by this point in the guide.
2. It could just as easily be a `computed` property or some other form of property. Given what's in `instance.md` I think this wording is fine.